### ISSUE TITLE
[CST] Update 5103 request so that Veteran sees a confirmation message when they submit the waiver

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#82580]
## Value Statement
**_As a_** veteran who just submitted a 5103 waiver
**_I want to_** to see a confirmation message telling me my submission was a success
**_So that_** I don't wonder whether my submission was actually received.

---
## Background Context

**This ticket is specifically scoped to the confirmation after the Veteran submits the 5103 waiver via the 5103 1P alert** 

This ticket is blocked by [82575](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82575/)

Current UI 

N/A

## Outcome, Success Measure, KPI(S), and Tracking Link
- More 5103 waivers submitted through CST
- More 5103 notices downloaded through claim letters section

## Design
- [Figma](https://www.figma.com/file/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation-and-5103-alert?type=design&node-id=2-11&mode=design)

For reference: 
<img width="396" alt="Screenshot 2024-05-07 at 10 37 57 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/5860a8fd-d616-4dfb-9ccd-965ea5fc6bc0">

### Enablement team (if needed)
@jstrothman

## Engineering 
Put behind feature flag `cst_5103_update_enabled`

## Out of scope
- 1P alert on Status and Files tab
- Document Request page for 5103 1P alert
- Recent Activity update

## Open questions
- 

## Tasks
Design:
- [ ] Finalize language for confirmation
- [ ] Finalize mock showing where this confirmation shows up for the Veteran

Eng:
- [ ] When a user submits the 5103 Notice Response, they are directed to the files page and see the 5103 waiver confirmation alert
- [ ] 5103 waiver confirmation is not stagnant and will be removed upon page refresh
- [ ] Put behind feature flag `cst_5103_update_enabled`

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria

- [ ] When a user submits the 5103 Notice Response, they are directed to the files page and see the 5103 confirmation alert
- [ ] If a user refreshes the page or navigate away and come back to it, the alert disappears
- [ ] Put behind feature flag `cst_5103_update_enabled`

**Comment by pmclaren19** [comment]
@skylerschain noticing this new green alert doesnt have the x button  it. Currently when you submit the 'Ask for tyour claim decision the green alert does have the x button. Is the intentional or do we want to add that so that users can remove the alert if they want?
